cmd/syncer/main.go

```diff
@@ -76,8 +76,9 @@ func main() {
 	if err != nil {
 		log.Errorf("Failed retrieving cluster flavor. Error: %v", err)
 	}
+	serviceMode := os.Getenv("X_CSI_MODE")
```

Comment:
I dont think we set this env variable for syncer

Reply:
Removed it and added an empty string to the function call